src/components/GameBoard.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+const GameBoard = ({ cards, handleCardClick }) => {
+  const suffleCards = (cards) => {
```

Comment:
It's best to shuffle the cards before passing them to the component, instead of shuffling them inside the component on each render. This is because shuffling the cards every time the component is rendered is unnecessary and can lead to performance issues. You can shuffle the cards outside the component and pass the shuffled cards as props.